plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/tasks/KtLintCheckTask.kt

```diff
@@ -38,7 +42,19 @@ open class KtlintCheckTask @Inject constructor(
         } else {
             logger.debug("Files changed: $filesToLint")
 
-            runLint(filesToLint)
+            runLint(filesToLint, false)
         }
     }
+
+    companion object {
```

Comment:
Either make this internal, or make the sub methods internal.

```suggestion
    internal companion object {
```

Reply:
Fixed via 5ece5b85